test/functional/verify/latest-missing/test.bats

```diff
@@ -5,7 +5,7 @@ load "../../swupdlib"
 @test "verify install using latest with missing version file on server" {
   run sudo sh -c "$SWUPD verify $SWUPD_OPTS --install -m latest"
 
-  check_lines "$output"
+  check_lines "$(echo "$output" | uniq)"
```

Comment:
Why pipe to uniq? I must be missing something.

Reply:
Swupd will try to download the version file using each one of the ca paths informed in compile time and we are going to have one curl error for each one of them. When using uniq the test will work no matter how many ca paths were used.